qsimcirq/qsim_simulator.py

```diff
@@ -118,14 +118,8 @@ def _sample_measure_results(
     if not isinstance(program, qsimc.QSimCircuit):
       program = qsimc.QSimCircuit(program, device=program.device)
 
-    if program.are_all_measurements_terminal() and repetitions > 1:
-      print('Provided circuit has no intermediate measurements. ' +
-            'It may be faster to sample from the final state vector. ' +
-            'Continuing with one-by-one sampling.')
-
     # Compute indices of measured qubits
     ordered_qubits = ops.QubitOrder.DEFAULT.order_for(program.all_qubits())
-    ordered_qubits = list(reversed(ordered_qubits))
```

Comment:
Why this line is not required?

Reply:
The state vector returned by `qsim_simulate_fullstate` is also in qsim (reverse) ordering, so the two cancel out. `_sample_measure_results`, has similar behavior.

(Note that `ordered_qubits` and `qubit_map` are only used by the all-terminal-measurements case)